### DISCUSSION TITLE
Oracle Cloud OAuth

### DISCUSSION THREAD

**Discussion by hashg** [directus/directus#14202] [Archive]
When integrating with Oracle Cloud OAuth - https://docs.oracle.com/en/cloud/paas/identity-cloud/rest-api/api-oauth-runtime-authorize-flow-browser-flow.html

Directus login is not redirecting correctly.

**Comment by rijkvanzanten** [comment]
Is that shared link referring to OAuth instead of OAuth2? If so, make sure to use OAuth2 or OpenID, as Directus doesn't support OAuth (v1) 👍🏻

**Comment by rijkvanzanten** [comment]
Also, if you run into what you believe is a bug, please describe in as much detail as possible what the issue is, and more importantly how to reproduce the issue. "Directus login is not redirecting correctly." doesn't mean anything out of context, and we have no idea what you configured / how we can test it out based on just the shared link 🙂
